crawl-ref/source/webserver/README.md

```diff
@@ -0,0 +1,85 @@
+# Webtiles server
+
+This is the Webtiles server for Dungeon Crawl Stone Soup.
+
+It's a HTTP server that allows users to play DCSS in a web browser.
+
+You can use it for small, personal setups or for large public servers.
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Running the server for testing purposes](#running-the-server-for-testing-purposes)
+* [Running a production server](#running-a-production-server)
+* [Contributing](#contributing)
+
+## Prerequisites
+
+To run the server, you need:
+
+* Linux or macOS (other platforms are untested)
+* Python 2.6 or newer (Python 3.6+ is preferred)
```

Comment:
I'm surprised the server works with python 2.6 but even if it does, we should definitely say python 2.7 here as the lower bound.